zarr/core.py

```diff
@@ -355,7 +356,7 @@ def _cdata_shape(self):
         if self._shape == ():
             return 1,
         else:
-            return tuple(int(np.ceil(s / c))
+            return tuple(int(math.ceil(s / c))
```

Comment:
Yeah I think you are right. Just wanted to check that this was the case for older Python versions as well.

Reply:
Looks like this has been the case for [all Python 3 versions]( https://docs.python.org/3.0/library/math.html#math.ceil ). So should be fine (since we are Python 3 only here).